### ISSUE TITLE
Not Finding Any Torrents

### ISSUE THREAD

**Issue by backslash05** [CouchPotato/CouchPotatoServer#6211]
### Steps to reproduce:
1. Add Movie
2. Refresh and check logs

### Information:
Movie(s) I have this with: currently Star Wars: The Force Awakens previous problems with other tittles
Quality of the movie being searched: Best
Providers I use: IPT and TorrentDay
Version of CouchPotato: git:(CouchPotato:CouchPotatoServer master) c4fad95c (2016-02-24 14:05:59)
Running on: Linux

### Logs:
https://gist.github.com/backslash05/aa9794ab26f4b5e305f9

**Comment by CouchPotatoBot** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [x] Fill in all the dots. They're there for a reason
- [x] Post logs, either inline (for smaller logs) or using [Pastebin](http://pastebin.com/)
- [ ] Post full version information

Please make sure you also read [contribution guide](https://github.com/RuudBurger/CouchPotatoServer/blob/develop/contributing.md#issues) and followed all the steps. 
Make the title describe your issue. Having "CP not working" or "I get this bug" for 100 issues, isn't really helpful. My master will close issues if there isn't enough information. On a good day he will tag the issue on close with the reason (like `can't reproduce`), but usually he won't, the lazy asshat.

Sometimes my master seems like a grumpy cat and responds with short answers. This isn't (always) because he hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let him know. Maybe he can teach me his awesome ways.

Thanks!

**Comment by backslash05** [comment]
I noticed that all torrents searched include " before and after... so I tried manual searches and both IPT and TorrentDay and found that it finds no torrents when you include the " and finds the torrents fine without the ". I have tried to figure out how I could remove them in the code to test my theory but I am not experienced at all in coding, so no luck finding it. But I do believe that this is the cause of the problem. Honestly CP hasn't been working for me in about a month, tried everything to no avail... so maybe this will fix my problem?
